src/views/deployments/DeploymentTable.tsx

```diff
@@ -17,7 +17,7 @@ const headers = ['Image pull connection ID', 'Image', 'Replicas', 'State', 'Crea
 const extractRow = (md: ModelDeployment) => [
     (
         <RouterLink
-            key="toolchain"
+            key="trainingIntegration"
```

Comment:
```suggestion
            key="imagePullConnIDLink"
```
Looks like it's more close to real meaning of it.